Gruntfile.js

```diff
@@ -16,13 +16,25 @@ var _ = require("lodash");
 var path = require("path");
 var execSync = require("child_process").execSync;
 
+var getFromExec = function (command) {
+    var result;
+    try {
+        result = execSync(command);
+    } catch (e) {
+        console.log("Error executing command: " + command);
```

Comment:
The error and stack trace output that appears from this task is unsightly and will probably confuse users who are in this workflow. I suggest that we add an "options" block to this function with a configuration silent: true/false which will swallow this output conditionally. We can then perhaps also support a default return value in this options block.